--- a/usr/share/alsa/ucm2/Qualcomm/x1e80100/x1e80100.conf
+++ b/usr/share/alsa/ucm2/Qualcomm/x1e80100/x1e80100.conf
@@ -20,3 +20,12 @@ If.LENOVOSlim7x {
 	}
 	True.Include.7x.File "/Qualcomm/x1e80100/LENOVO-Slim-7x.conf"
 }
+
+If.ASUSProArtPZ13 {
+	Condition {
+		Type RegexMatch
+		String "${var:DMI_info}"
+		Regex "ASUSTeK COMPUTER.*ProArt PZ13"
+	}
+	True.Include.pz13.File "/Qualcomm/x1e80100/ASUS-ProArt-PZ13.conf"
+}
